kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/integrations/mikkelinyt/MikkeliNytEventProvider.java

```diff
@@ -0,0 +1,360 @@
+          InputStream data = IOUtils.toBufferedInputStream(response.getEntity().getContent());
+          String type = response.getEntity().getContentType().getValue();
+          return new Response<>(statusCode, message, new AttachmentData(type, data));
+        } finally {
+          response.close();
+        }
+      } finally {
+        imageClient.close();
+      }
+    } catch (IOException e) {
+      logger.log(Level.WARNING, String.format("Failed to fetch event imagae from %s", imageUrl), e);
+      return new Response<>(500, "Interval Server Error", null);
+    }
+  }
+  
+  private AttachmentId getImageAttachmentId(String url) {
+    String imageId = StringUtils.substringAfterLast(url, "/");
+    AttachmentId mikkeliNytId = new AttachmentId(MikkeliNytConsts.IDENTIFIER_NAME, imageId);
+    AttachmentId kuntaApiId = idController.translateAttachmentId(mikkeliNytId, KuntaApiConsts.IDENTIFIER_NAME);
+    
+    if (kuntaApiId == null) {
+      logger.info(String.format("Found new MikkeliNyt attachment %s", imageId));
+      Identifier newIdentifier = identifierController.createIdentifier(IdType.ATTACHMENT, MikkeliNytConsts.IDENTIFIER_NAME, imageId);
+      kuntaApiId = new AttachmentId(KuntaApiConsts.IDENTIFIER_NAME, newIdentifier.getKuntaApiId());
+    }
+    
+    return kuntaApiId;
+  }
+
+  private List<Event> transform(List<fi.otavanopisto.mikkelinyt.model.Event> nytEvents) {
+    List<Event> result = new ArrayList<>(nytEvents.size());
+    
+    for (fi.otavanopisto.mikkelinyt.model.Event nytEvent : nytEvents) {
+      result.add(transform(nytEvent));
+    }
+    
+    return result;
+  }
+  
+  private Event transform(fi.otavanopisto.mikkelinyt.model.Event nytEvent) {
+    if (nytEvent == null) {
+      return null;
+    }
+    
+    EventId mikkeliNytId = new EventId(MikkeliNytConsts.IDENTIFIER_NAME, nytEvent.getId());
+    EventId kuntaApiId = idController.translateEventId(mikkeliNytId, KuntaApiConsts.IDENTIFIER_NAME);
+    if (kuntaApiId == null) {
+      logger.info(String.format("Found new MikkeliNyt event %s", nytEvent.getId()));
+      Identifier newIdentifier = identifierController.createIdentifier(IdType.EVENT, MikkeliNytConsts.IDENTIFIER_NAME, nytEvent.getId());
+      kuntaApiId = new EventId(KuntaApiConsts.IDENTIFIER_NAME, newIdentifier.getKuntaApiId());
+    }
+    
+    Event result = new Event();
+    
+    result.setAddress(stripHtml(nytEvent.getAddress()));
+    result.setCity(stripHtml(nytEvent.getCity()));
+    result.setDescription(nytEvent.getDescription());
+    result.setEnd(parseOffsetDateTime(nytEvent.getEnd()));
+    result.setId(kuntaApiId.getId());
+    result.setName(stripHtml(nytEvent.getName()));
+    result.setOriginalUrl(nytEvent.getUrl());
+    result.setPlace(stripHtml(nytEvent.getPlace()));
+    result.setZip(stripHtml(nytEvent.getZip()));
+    result.setStart(parseOffsetDateTime(nytEvent.getStart()));
+    
+    return result;
+  }
+  
+  private static String stripHtml(String html) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "stripHtml" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)